imports/ui/components/review/review.js

```diff
@@ -62,7 +62,14 @@ Template.review.helpers({
       applicant.scores = [];
       applicant.weight = 0;
       applicant.notes = [];
+      applicant.interviewer_name = [];
       _.each(interviewings, function(i) {
+        const usernameToPush = Meteor.users.findOne({ "emails": { $elemMatch: { "address": i.user_email, "verified":false } } }).username
```

Comment:
* Style: keys in JS objects (dictionaries) don't need to be quoted.
* I don't think it's necessary to include the `verified` key in the query, you can double check in `meteor mongo` to see if it still users by email.